style.css

```diff
@@ -389,4 +389,16 @@ footer #email:hover {
     width: 70px;
   }
 
+}
+.container-glass {
```

Comment:
We can also add a small `Border radius` - it'll give a premium look.
Plus, your branch still has conflicts , use `git pull origin <origin branch name>` to pull and resolve all the conflicts.

Reply:
Changes done.
<img width="402" alt="image" src="https://user-images.githubusercontent.com/47862474/194246444-a05dc5c4-3c93-4182-862d-37b9930a2d35.png">